### PR TITLE
fix(render):headers set fail

### PR DESCRIPTION
**Pull request by mouuii**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: runtime/render.go [hunk 1]
```diff
@@ -246,16 +246,16 @@ func renderDefinition(ctx *Context, dcs []types.Definition) ([]Definition, error
 	return ds, nil
 }
 
-func renderHeader(ctx *Context, current map[string]string, headers map[string]types.Template) error {
-	if current == nil {
-		current = map[string]string{}
+func renderHeader(ctx *Context, current *map[string]string, headers map[string]types.Template) error {
```
Comment (liubog2008) [inline runtime/render.go]: please change to

```
func renderHeader(ctx *Context, current map[string]string, headers map[string]types.Template) (map[string]string, error)
```

**Comment by liubog2008** [comment]
Thanks for your PR, could you fix your changes?

File: runtime/render.go (hunk 1 above)
Comment (mouuii) [reply runtime/render.go]: changed

**Comment by mouuii** [comment]
fixed @liubog2008

**Pull request merged**